tf/examples/Bonsai/bonsai_example.py

```diff
@@ -10,101 +11,113 @@
 from edgeml.trainer.bonsaiTrainer import BonsaiTrainer
 from edgeml.graph.bonsai import Bonsai
 
-
 def main():
-    # Fixing seeds for reproducibility
-    tf.set_random_seed(42)
-    np.random.seed(42)
+	# Fixing seeds for reproducibility
+	tf.set_random_seed(42)
+	np.random.seed(42)
+
+	# Hyper Param pre-processing
+	args = helpermethods.getArgs()
+	isRegression = None
 
-    # Hyper Param pre-processing
-    args = helpermethods.getArgs()
+	# The only argument that needs to be changed to change between regression and classification is , 'isRegression'.
+	if (args.regression is not None):
+	    isRegression = args.regression
+	else:
+	    isRegression = False
 
-    sigma = args.sigma
-    depth = args.depth
+	sigma = args.sigma
+	depth = args.depth
 
-    projectionDimension = args.proj_dim
-    regZ = args.rZ
-    regT = args.rT
-    regW = args.rW
-    regV = args.rV
+	projectionDimension = args.proj_dim
+	regZ = args.rZ
+	regT = args.rT
+	regW = args.rW
+	regV = args.rV
 
-    totalEpochs = args.epochs
+	totalEpochs = args.epochs
 
-    learningRate = args.learning_rate
+	learningRate = args.learning_rate
 
-    dataDir = args.data_dir
+	data_dir = args.data_dir
 
-    outFile = args.output_file
+	outFile = args.output_file
+	print ("outFile : ",outFile)
 
-    (dataDimension, numClasses,
-        Xtrain, Ytrain, Xtest, Ytest) = helpermethods.preProcessData(dataDir)
+	(dataDimension, numClasses,
+	    Xtrain, Ytrain, Xtest, Ytest) = helpermethods.preProcessData(data_dir,isRegression)
 
-    sparZ = args.sZ
+	sparZ = args.sZ
 
-    if numClasses > 2:
-        sparW = 0.2
-        sparV = 0.2
-        sparT = 0.2
-    else:
-        sparW = 1
-        sparV = 1
-        sparT = 1
+	if numClasses > 2:
```

Comment:
Yeah, I think these diffs are due to tabs in place of spaces. Can you convert all your tabs to spaces.